example-app/src/app/local-state/local-state.reducer.ts

```diff
@@ -0,0 +1,74 @@
+import { Action, combineReducers } from '@ngrx/store';
+import { createFormGroupState, formGroupReducer, FormGroupState, updateGroup, setValue } from 'ngrx-forms';
+
+export class GetManufacturersAction implements Action {
+  static readonly TYPE = 'localState/GET_MANUFACTURERS';
+  readonly type = GetManufacturersAction.TYPE;
+  constructor(public countryCode: string) { }
+}
+
+export class SetManufacturersAction implements Action {
+  static readonly TYPE = 'localState/SET_MANUFACTURERS';
+  readonly type = SetManufacturersAction.TYPE;
+  constructor(public manufacturers: string[]) { }
+}
+
+export interface FormValue {
+  countryCode: string;
+  manufacturer: string;
+}
+
+export interface LocalState {
+  manufacturers: string[];
+  formState: FormGroupState<FormValue>;
+}
+
+export const FORM_ID = 'localStateForm';
+
+export const INITIAL_FORM_STATE = createFormGroupState<FormValue>(FORM_ID, {
+  countryCode: '',
+  manufacturer: ''
+});
+
+export const INITIAL_LOCAL_STATE: LocalState = {
+  manufacturers: [],
+  formState: INITIAL_FORM_STATE
+};
+
+const reducers = combineReducers<LocalState>({
+  manufacturers(manufacturers = [], a: Action) {
+    // update from loaded data
+    if (a instanceof SetManufacturersAction) {
```

Comment:
personally I never use `instanceof` with actions, but instead compare the `type`. This is because it is not guaranteed the action will always be a class instance (e.g. if dispatched via the redux devtools).